test/sync_test_case.rb

```
@@ -0,0 +1,76 @@
+
+  def create_sync_status(...)
+    TestFactories::SyncStatus.create(...)
+  end
+
+  def sample_article(...)
+    TestFactories::Article.build_details(...)
+  end
+
+
+  def create_app(**)
+    App.new(http_client: @http_client, **)
```

RuboCop (1.69.1)

[fatal] unexpected token tRPAREN
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)